### ISSUE TITLE
Deprecate Daily Data Test

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/datastore#121]

**Comment by KDuerden** [comment]
Can you confirm is deprecating the DDT would have on the data available to the internal Dashboard - this is used for our funder and board monitoring, and so needs to be maintained.

**Comment by mariongalley** [comment]
Yes! We won't remove of change anything without talking to the publisher support team.
